lib/cuckoo/core/database.py

```diff
@@ -835,9 +835,26 @@ def _task_arch_tags_helper(self, task: Task):
 
         return task_archs, task_tags
 
+    def validate_tasks_parameters(self,label,platform,tags) -> bool:
```

Comment:
This is a nitpick, but I think it is important to note that this method is only used for the parameters of a single task
```suggestion
    def validate_task_parameters(self,label,platform,tags) -> bool:
```